playbooks/collect_activity/collect_activity.yml

```diff
@@ -0,0 +1,37 @@
+---
+# A simple one-off playbook that gathers basic usage information from OpenedX hosts.
+# This is used by the `activity_csv` management command.
+
+- hosts: all
+  vars:
+    # This should be overridden to be more unique.
+    local_output_dir: /tmp/
+    remote_output_filename: /tmp/activity_report
+    section_name: '{{ inventory_hostname }}'
```

Comment:
Yeah, this isn't really necessary to set here. I thought it would make things clearer but it really doesn't - so I changed it to just directly pass the hostname to the command in the step below as you mentioned.